packages/arb-token-bridge-ui/src/pages/api/cctp/[type].ts

```diff
@@ -0,0 +1,211 @@
+import {
+  ApolloClient,
+  ApolloQueryResult,
+  gql,
+  HttpLink,
+  InMemoryCache
+} from '@apollo/client'
+import { NextApiRequest, NextApiResponse } from 'next'
+import { ChainId } from '../../../util/networks'
+
+export function getSubgraphClient(subgraph: string) {
+  return new ApolloClient({
+    link: new HttpLink({
+      uri: `https://api.thegraph.com/subgraphs/name/chrstph-dvx/${subgraph}`,
+      fetch
+    }),
+    cache: new InMemoryCache()
+  })
+}
+
+// Extending the standard NextJs request with CCTP params
+export type NextApiRequestWithCCTPParams = NextApiRequest & {
+  query: {
+    address: `0x${string}`
+    l1ChainId: ChainId
+  }
+}
+
+export type MessageReceived = {
+  blockNumber: number
+  blockTimestamp: number
+  caller: `0x${string}`
+  id: string
+  messageBody: string
+  nonce: number
+  sender: `0x${string}`
+  sourceDomain: 0 | 1 // 0: Mainnet, 1: Avalanche
```

Comment:
Hm why do we list out Avalanche here?

Reply:
It was a leftover, I replaced it with `ChainDomain`